src/applications/vaos/services/appointment/transformers.js

```diff
@@ -501,8 +509,8 @@ export function transformPendingAppointment(appt) {
     id: appt.id,
     status: getRequestStatus(appt, isExpressCare),
     created,
-    cancelationReason: unableToReachVeteran
-      ? UNABLE_TO_REACH_VETERAN_DETCODE
+    cancellationReason: unableToReachVeteran
```

Comment:
We could just set this to null. Doesn't look like it's used for requests now (I think it was for Express Care). Setting it to null would be easier to keep consistent with v2.